### DISCUSSION TITLE
Make plugin also work for .html files

### DISCUSSION THREAD

**Discussion by ronaldhoek** [gund/eslint-plugin-deprecation#66] [Ideas]
Love this plugin - thanks!

I was wondering whether the plugin could be easily adjusted to make it also work in the .html files?
Since I don't have any experience building lint plugins, I don't have any clue if this is even possible...

**Comment by gund** [comment]
I guess your question is not related to this plugin but in general how to run Eslint on html files.

[Google](https://www.google.com/search?q=eslint+html+files&oq=eslint+html+files&aqs=chrome..69i57j69i60l2j69i61.4861j0j7&sourceid=chrome&ie=UTF-8) (or chatgpt) is your friend here.

**Reply by ronaldhoek** [reply]
Thanks for the reply @gund 
I tried activating the plugin for the HTML files, the same way as I did for the .ts files - however, the rule does not seem to work for .html only for .ts files...

**Reply by gund** [reply]
Would be helpful if you could create a repro case either online or a repo so that we can check it out.

**Reply by ronaldhoek** [reply]
See: https://github.com/ronaldhoek/ESLint-HTML-deprecation-test

Linting in TS file works
![image](https://github.com/gund/eslint-plugin-deprecation/assets/5037087/2f1e5985-4711-4931-b3ab-21d08e8f05a8)

But would love to see the same in the HTML file; ;)
![image](https://github.com/gund/eslint-plugin-deprecation/assets/5037087/e0668ef9-f8fd-45da-8667-461f9cc76ad2)
